superset-frontend/src/dashboard/actions/hydrate.js

```diff
@@ -366,7 +366,8 @@ export const hydrateDashboard = (dashboardData, chartData) => (
         directPathLastUpdated: Date.now(),
         focusedFilterField: null,
         expandedSlices: metadata?.expanded_slices || {},
-        refreshFrequency: metadata?.refresh_frequency || 0,
+        refreshFrequency:
+          metadata?.refresh_frequency || dashboardState.refreshFrequency || 0,
```

Comment:
I'm not sure about the `|| dashboardState.refreshFrequency` here. If you start at dashboard A (which has refresh), navigate to the dashboard list and then go to dashboard B (which has no refresh) wouldn't B inherit the refresh frequency from A?

Reply:
I wish we had a redux store where these things were keyed by dashboard id

Reply:
@suddjian You are totally right! I tested and B is inheriting from A. I'll try to fix it.